tests/index.js

```diff
@@ -3,31 +3,29 @@ var hpkp = require('../lib/hpkp')
 var Hapi = require('hapi')
 var server
 
-function createServer(port, hpkpOptions) {
-  server = new Hapi.Server()
-  server.connection({
+async function createServer(port, hpkpOptions) {
+  server = new Hapi.Server({
     port: port
   })
 
-  server.register({
-    register: require('../index.js'),
-    options: hpkpOptions
-  }, function (err) {
-    if (err) {
-      console.error('Failed to load plugin:', err)
-    }
-  });
+  try{
+   await server.register({
+      plugin: require('../index.js'),
+      options: hpkpOptions
+    })
+  }catch(err){
+    console.error('Failed to load plugin:', err)
+  }
 
-  server.route({
+  await server.route({
     method: 'GET',
     path: '/',
     handler: function (request, reply) {
```

Comment:
should the wait be before `server.route` or before the function here in the handler? `handler: await function`